pkg/provisioner/ironic/validatemanagementaccess_test.go

```diff
@@ -412,7 +416,7 @@ func TestValidateManagementAccessExistingSteadyStateNoUpdate(t *testing.T) {
 			}
 			assert.Equal(t, "", result.ErrorMessage)
 			assert.Equal(t, false, result.Dirty)
-			assert.Len(t, ironic.GetLastNodeUpdateRequestFor("uuid"), 0)
+			assert.Len(t, ironic.GetLastNodeUpdateRequestFor("uuid"), imageType.ExpectedUpdates)
```

Comment:
The existence of this test suggests that we explicitly don't want to update nodes in "ValidateManagementAccess",
yet here we are, is this ok?

Reply:
I think probably to avoid this, we should add the `DriverInfo` (when creating the mock nodes in `testserver.NewIronic(t).Ready().CreateNode` above?

That way the node will already be in the expected state, e.g the steady-state-no-update case this test is trying to prove, and no PATCH update will be triggered?

Reply:
makes sense, fixed it, thanks